projects/state of rust adoption survey/2025/proposed_questions.md

```diff
@@ -0,0 +1,133 @@
+Some Proposed Questions
+
+### Prelude
+
+Do you work in a safety critical industry? (Automotive, aerospace, medical, robotics, etc)
+ *Select one*
+
+- [ ] Yes
+- [ ] No
+
+What industry do you work in?
+ *Select multiple*
+
+- [ ] Automotive
+- [ ] Aerospace
+- [ ] Medical
+- [ ] Industrial
+- [ ] Robotics
+- [ ] Defense
+- [ ] Other (text entry)
+
+What is your primary role or responsibility related to software development or safety within your organization?
+
+- [ ] Software Engineer
+- [ ] Safety Engineer
+- [ ] Quality Assurance Engineer
+- [ ] Project Manager
+- [ ] R&D Engineer
+- [ ] Engineering Management
+- [ ] Other (text entry)
+
+Do you currently use Rust?
+ *Select multiple*
+
+- [ ] Yes, in my safety critical role
+- [ ] Yes, as a hobby/in my free time
+- [ ] No
+
+What language(s) do you use in your safety critical role?
+ *Select multiple*
+
+- [ ] Rust
+- [ ] C
+- [ ] C++
+- [ ] Ada
+- [ ] Python
+- [ ] Java
+- [ ] Other
```

Comment:
With "Other", should we have a text box that allows for free-form input of a language?